### ISSUE TITLE
AWS.Logger.AspNetCore: Do the stack traces for exceptions go anywhere?

### ISSUE THREAD

**Issue by CL0SeY** [aws/aws-logging-dotnet#3]
I'm using this package to send Warning + higher level messages - but the actual detail of exceptions don't seem to go anywhere.

Is this by design or something that can be configured?

**Comment by spati2** [comment]
Hi,

Could you give an example of the message that you are seeing on Cloudwatch vs what is expected?

**Comment by golfguy0082** [comment]
I'm having a similar problem.  I believe that I'm having a permissions issue, but I can't tell for sure if I can't see the stack trace.  It looks like exceptions are being swallowed when in release mode: https://github.com/aws/aws-logging-dotnet/blob/master/src/AWS.Logger.Core/Core/AWSLoggerCore.cs#L219-L228

Would be nice to write these to System.Diagnostics.Trace instead so we could catch these issues.  Log4net does the same: http://logging.apache.org/log4net/release/faq.html#internalDebug

**Comment by spati2** [comment]
Makes sense, I will add a mechanism to catch the exception message there. Thanks for pointing that out

**Comment by MamdouhM** [comment]
I am Facing the same issue @CL0SeY  mentioned.
@spati2 : when do you plan to fix this?

**Comment by spati2** [comment]
Added a log file to capture any exceptions raised in AWS.Logger.Core package.

**Comment by spati2** [comment]
Resolving issue after awaiting response for 7 days

**Comment by iamoses** [comment]
Is the "aws-logger-errors.txt" file referenced in any documentation?
I looked in the acw-ug.pdf documentation for CloudWatch, to no avail.
The reason I'm asking is that I had configured my .NET Core project to 
log to CloudWatch, which it was doing.  However; it was also blowing up
as follows: 
`Exception caught when writing error log to fileSystem.IO.IOException: Read-only file system`

After searching through this repository and its history, I saw the default setting for the AWSLoggerConfig class as follows:
`public string LibraryLogFileName { get; set; } = "aws-logger-errors.txt";`
